Source/CarthageKit/FrameworkExtensions.swift

```diff
@@ -322,27 +326,25 @@ extension Reactive where Base: FileManager {
 
 			var temporaryDirectoryTemplate: ContiguousArray<CChar> = temporaryDirectory.appendingPathComponent(template).utf8CString
 
-			let result: UnsafeMutablePointer<Int8>? = temporaryDirectoryTemplate.withUnsafeMutableBufferPointer { (template: inout UnsafeMutableBufferPointer<CChar>) -> UnsafeMutablePointer<CChar> in
-				return mkdtemp(template.baseAddress)
+			let result: UnsafeMutablePointer<Int8>? = temporaryDirectoryTemplate
+			.withUnsafeMutableBufferPointer { (template: inout UnsafeMutableBufferPointer<CChar>) -> UnsafeMutablePointer<CChar> in
+				mkdtemp(template.baseAddress)
```

Comment:
You mean the entire `.swithUnsafeMutableBufferPointer` has to be indented with one more tab? Done.